pkgs/applications/misc/iterm2/default.nix

```diff
@@ -1,18 +1,22 @@
-{ stdenv, fetchurl, unzip }:
+{ stdenv, fetchFromGitHub, unzip }:
 
 stdenv.mkDerivation rec {
   name = "iterm2-${version}";
-  version = "2.1.4";
+  version = "3.0.2";
 
-  src = fetchurl {
-    url = "https://iterm2.com/downloads/stable/iTerm2-2_1_4.zip";
-    sha256 = "1kb4j1p1kxj9dcsd34709bm2870ffzpq6jig6q9ixp08g0zbhqhh";
+  src = fetchFromGitHub {
+    owner = "gnachman";
+    repo = "iTerm2";
+    rev = "refs/tags/v${version}";
+    name = "iTerm2-${version}";
```

Comment:
You can actually just set "rev" to the tag you need:

```nix
rev = "v${version}";
```

and no "name" is necessary.